tasks/request_from_archive/request_from_archive.py

```diff
@@ -183,35 +185,40 @@ def get_glacier_recovery_type(config: Dict[str, Any]) -> str:
             )
 
     # Look for default from TF
+    recovery_type = os.getenv(OS_ENVIRON_DEFAULT_RECOVERY_TYPE_KEY, None)
     if recovery_type is None:
-        recovery_type = os.getenv(OS_ENVIRON_DEFAULT_RECOVERY_TYPE_KEY, None)
-        if recovery_type is not None:
-            if recovery_type in VALID_RESTORE_TYPES:
-                LOGGER.info(
-                    f"Using restore type of {recovery_type} "
-                    f"found in environment variable {OS_ENVIRON_DEFAULT_RECOVERY_TYPE_KEY}."
-                )
-            else:
-                LOGGER.error(
-                    f"Invalid restore type value of '{recovery_type}' "
-                    f"found in environment variable {OS_ENVIRON_DEFAULT_RECOVERY_TYPE_KEY}."
-                )
-                raise ValueError(
-                    f"Invalid restore type value in environment variable "
-                    f"{OS_ENVIRON_DEFAULT_RECOVERY_TYPE_KEY}"
-                )
+        recovery_type = "Standard"
```

Comment:
Default now set here. Since TF will always set the os.environ default, this did not affect customers.